Modules/Microsoft365DSC/DSCResources/MSFT_IntuneWindowsUpdateForBusinessFeatureUpdateProfileWindows10/MSFT_IntuneWindowsUpdateForBusinessFeatureUpdateProfileWindows10.psm1

```diff
@@ -0,0 +1,1375 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        #region resource generator code
+        [Parameter()]
+        [System.String]
+        $Description,
+
+        [Parameter()]
+        [System.String]
+        $DisplayName,
+
+        [Parameter()]
+        [System.String]
+        $FeatureUpdateVersion,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance]
+        $RolloutSettings,
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Id,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance[]]
+        $Assignments,
+        #endregion
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        [ValidateSet('Absent', 'Present')]
+        $Ensure = $true,
```

Comment:
Incorrect default value: Should be 'Present' instead of $true. This is probably part of the DRG as well, since I had the same issue with the resource I am working on as well.

Should be corrected for the Set and Test method as well.